incorrect docs for excinfo.group_contains

https://docs.pytest.org/en/stable/how-to/assert.html#assert-matching-exception-groups v: stable

gives the following as example:
```python
def test_exception_in_group():
    with pytest.raises(RuntimeError) as excinfo:
        raise ExceptionGroup(
            "Group message",
            [
                RuntimeError("Exception 123 raised"),
            ],
        )
    assert excinfo.group_contains(RuntimeError, match=r".* 123 .*")
    assert not excinfo.group_contains(TypeError)
```
running that code on pytest==8.0.0, python==3.11.6, Linux gives
```
$ pytest test_bar.py 
===================================== test session starts =====================================
platform linux -- Python 3.11.6, pytest-8.0.0, pluggy-1.4.0
rootdir: /tmp/pytest_8
collected 1 item                                                                              

test_bar.py F                                                                           [100%]

========================================== FAILURES ===========================================
___________________________________ test_exception_in_group ___________________________________
  + Exception Group Traceback (most recent call last):
...
  +-+---------------- 1 ----------------
    | RuntimeError: Exception 123 raised
    +------------------------------------
=================================== short test summary info ===================================
FAILED test_bar.py::test_exception_in_group - ExceptionGroup: Group message (1 sub-exception)
====================================== 1 failed in 0.01s ======================================
```


I'm pretty sure the example is supposed to be
```python
import pytest
def test_exception_in_group():
    with pytest.raises(ExceptionGroup) as excinfo:
        raise ExceptionGroup(
            "Group message",
            [
                RuntimeError("Exception 123 raised"),
            ],
        )
    assert excinfo.group_contains(RuntimeError, match=r".* 123 .*")
    assert not excinfo.group_contains(TypeError)
```
changing `pytest.raises(RuntimeError)` to `pytest.raises(ExceptionGroup)`

--- a/doc/en/how-to/assert.rst
+++ b/doc/en/how-to/assert.rst
@@ -154,7 +154,7 @@ method to test for exceptions returned as part of an :class:`ExceptionGroup`:
 .. code-block:: python
 
     def test_exception_in_group():
-        with pytest.raises(RuntimeError) as excinfo:
+        with pytest.raises(ExceptionGroup) as excinfo:
             raise ExceptionGroup(
                 "Group message",
                 [
@@ -176,7 +176,7 @@ exception at a specific level; exceptions contained directly in the top
 .. code-block:: python
 
     def test_exception_in_group_at_given_depth():
-        with pytest.raises(RuntimeError) as excinfo:
+        with pytest.raises(ExceptionGroup) as excinfo:
             raise ExceptionGroup(
                 "Group message",
                 [

